lib/migration.c

```diff
@@ -534,6 +534,60 @@ migration_region_access(vfu_ctx_t *vfu_ctx, char *buf, size_t count,
     return count;
 }
 
+bool
+migration_feature_supported(uint32_t flags) {
```

Comment:
migration.c shouldn't be dealing with DEVICE_FEATURE encoding: you should pass in the masked out flags section only.